### DISCUSSION TITLE
Should falsy frontmatter values defer to other values higher in the data cascade?

### DISCUSSION THREAD

**Discussion by silveltman** [11ty/eleventy#4164] [Enhancement Queue]
**Describe the bug**
11ty uses a frontmatter value instead of a global data file value, even though the frontmatter value is empty. 

**To Reproduce**

1. Go to create an 11ty project
2. create the data file and frontmatter below
3. build/serve 11ty
4. Look at build output

**Expected behavior**
The frontmatter should be ignored and the value in the data file should be used

**Environment:**
- Windows 10
- Eleventy Version 1.01

**Additional context**"
__data/seo.yaml_
```
title:
description:
```
_Frontmatter_
```
seo:
  title:
  description:
```

**Comment by zachleat** [comment]
HMMMM. I see what you’re saying. But I think the current behavior is the most technically accurate and has the least amount of magic (is the most predictable).

Should all falsy values defer up the cascade?
